src/App.js

```diff
@@ -14,7 +14,7 @@ class App extends React.Component {
 
 	render() {
 		return (
-			<div>
+			<div style={{height: 2000}}>
```

Comment:
Why is this needed?

Reply:
I needed the playground to scroll to reproduce the issue, and then I've just forgotten about remove it. This happens to me all the time with the playground :D

Reply:
Hey @onlined, do you want to be member of the project? After seeing your PRs and the detailed reviews you made I'd be glad to have you aboard. (I could have reviews on my PRs too!)

Reply:
It would be great, thank you very much!

Reply:
Nice1 I've sent an invite for you :)